src/System.IO.Packaging/src/System/IO/Packaging/IgnoreFlushAndCloseStream.cs

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+//------------------------------------------------------------------------------
+//  File:           IgnoreCloseAndFlushStream.cs
+//
+//  Description:    The class is used to wrap a given stream in a way that the Flush
+//                  and Close calls to the stream are Ignored. This stream class has been
+//                  created specifically for perf improvements for the ZipPackage.
+//------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+
+namespace System.IO.Packaging
+{
+    /// <summary>
+    /// This class ignores all calls to Flush() and Close() methods
+    /// depending on whether the IgnoreFlushAndClose property is set to true
```

Comment:
This comment appears out-of-date: there is no IgnoreFlushAndClose property, and Flush/Close never delegate to the wrapped stream's corresponding members.